test/testfolderstatusmodel.cpp

```diff
@@ -44,8 +45,103 @@ private Q_SLOTS:
 
         _folderMan.reset(new FolderMan{});
     }
+    
+    void testThatFolderManHasInstance() {
+        QVERIFY2(nullptr != FolderMan::instance(), "folder manager must not be a NULL");
+    }
+    
+    void testEmptyModelHasOneColumn()
+    {
+        const FolderStatusModel test;
+        QCOMPARE(test.columnCount(), 1);
+    }
+    
+    void testAccountStateIsConnected()
+    {
+        const auto fakeFolder = makeFakeFolder();
+        const FakeAccountState accountState{fakeFolder.account()};
+        QVERIFY2(accountState.isConnected(), "account must be connected");
+    }
```

Comment:
My impression is this test is out of place, we should be verifying the behaviour of this in a test of the FakeFolder/FakeAccountState